helix-lsp/src/lib.rs

```diff
@@ -410,6 +426,17 @@ impl LspProgressMap {
     }
 }
 
+fn load_lsp_config_file(language: &str) -> anyhow::Result<Option<serde_json::Value>> {
```

Comment:
Instead of extra files, what do you think about making this a `config` key under the language_server toml key? Then one can use

```toml
config = """
{
   ...
}
"""
```
to declare the config

Reply:
That was my first thought, (my second was to put it in the helix config file), but I decided against that because editing json in a toml file wasn't very ergonomic.  That said, I am not against it; I will make that chance.